app/src/main/java/com/taylorgirard/comicconvo/fragments/ProfileFragment.java

```diff
@@ -151,6 +152,10 @@ public void done(ParseException e) {
             @Override
             public void onClick(View v) {
                 ParseUser.logOutInBackground();
+                ParseInstallation installation = ParseInstallation.getCurrentInstallation();
+                ArrayList<String> channel = new ArrayList<>();
+                installation.put("channels", channel);
```

Comment:
This puts an empty channel for the current installation when the user logs out, so that the device no longer gets notifications meant to be sent to that user